sway-lib-std/src/external.sw

```diff
@@ -0,0 +1,13 @@
+library external;
+
+use ::contract_id::ContractId;
+
+/// Get the root of the bytecode of the contract at 'contract_id'.
+pub fn bytecode_root(contract_id: ContractId) -> b256 {
+    asm(buffer, target: contract_id.value) {
+        move buffer sp;
+        cfei i32;
+        croo buffer target;
+        buffer: b256
```

Comment:
Now that Sway has calls and call-frames we can't leak anything onto the stack any more.  All `cfei`s must be paired with `cfsi`s, or you use the heap.

In this case though I think you could create a local `b256`, take its address and populate it with `croo` and return that.

In the ongoing effort to remove ASM blocks from the libs, rather than introduce new ones, :wink: this could be made into a compiler intrinsic, like `__gtf`.